e2e/lvm/lvm_suite_test.go

```diff
@@ -33,3 +36,59 @@ var _ = Describe("Validation test", func() {
 		})
 	})
 })
+
+// Test to verify the PVC status
+var _ = Describe("PVC Status check", VerifyPVCStatus)
+
+func VerifyPVCStatus() {
+
+	Describe("pvc", func() {
+		var filepvc *k8sv1.PersistentVolumeClaim
+		var filepod *k8sv1.Pod
+		var blockpvc *k8sv1.PersistentVolumeClaim
+		var blockpod *k8sv1.Pod
+		var namespace string
+
+		BeforeEach(func() {
+			namespace = tests.TestNamespace
+			filepvc = tests.GetSamplePVC(tests.StorageClass, "5Gi", "lvmfilepvc", k8sv1.PersistentVolumeFilesystem)
+			filepod = tests.GetSamplePod("lvmfilepod", "lvmfilepvc")
+			blockpvc = tests.GetSamplePVC(tests.StorageClass, "5Gi", "lvmblockpvc", k8sv1.PersistentVolumeBlock)
+			blockpod = tests.GetSamplePod("lvmblockpod", "lvmblockpvc")
+		})
+
+		AfterEach(func() {
+			err := tests.DeployManagerObj.GetK8sClient().CoreV1().Pods(namespace).Delete(context.TODO(), filepod.Name, metav1.DeleteOptions{})
```

Comment:
- if we are going to use `k8sClient` multiple times in multiple places why not register that in `BeforeSuite` (create a `var` at pkg level) if `SuiteFailed` to `False` and use that throughout?
- iirc there was also an effort to unify all clients used in the test and having that at pkg level might be helpful?

Reply:
Yes, will unify all clients in the next PR.